ships/deprecate-runtime.md

```diff
@@ -0,0 +1,125 @@
+<!--
+Copyright The Shipwright Contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+---
+title: deprecate-runtime
+authors:
+  - "@imjasonh"
+reviewers:
+  - "@adambkaplan"
+  - "@qu1queee"
+approvers:
+  - "@adambkaplan"
+  - "@qu1queee"
+creation-date: 2021-05-26
+last-updated: 2021-05-26
+status: implementable
+replaces:
+  - "https://github.com/shipwright-io/build/blob/main/docs/proposals/runtime-image.md"
+---
+
+# Deprecate `runtime`
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [x] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [docs](/docs/)
+
+## Open Questions [optional]
+
+What features of `runtime` do we expect future users to want, and how should we deliver them in a manner that's consistent with the future direction of the API?
```

Comment:
Should we mark this SHIP as "provisional" as long as this question is open?

Reply:
Good question. I would bias toward action here, even if there are unanswered questions. While the feature is live and non-deprecated, we risk people discovering it and coming to depend on it, which will make deprecating and removing harder later.

We can delay removing it for as long as we feel necessary, but we should deprecate ASAP if we think we ever will (and I believe we have consensus on that much at least).